OAK-Marketplace-Emotion-Recognition/utils/inference.py

```diff
@@ -0,0 +1,25 @@
+from emotion_recognition_retail import process_frame
```

Comment:
There is no such method. You probably implemented it locally. 

`from oak_inference_utils.inference import process_frame` is the correct import